### ISSUE TITLE
3D-Touch submission action issue

### ISSUE THREAD

**Issue by HyperBCS** [ccrama/Slide-iOS#1010]
When setting 3D-Touch submission action to "Peek Content" it defaults to haptic touch(long press) and opens the Submission Menu. This behavior does not happen on other options for 3D-Touch submission action.

**Comment by ccrama** [comment]
Hello, are you seeing this behavior on Images and Gifs from the main submission page?

**Comment by HyperBCS** [comment]
Yes. The behavior occurs on the main submission page as well as any subreddit pages. My phone is an iPhone Xs Max on iOS 13.5

**Comment by HyperBCS** [comment]
I'm not sure why this is the default behavior, but it seems "Peek Content" does work, but it is enabled only for the frame of the preview in the LinkCellView. In my opinion it would make more sense to remove the condition for checking if the menu interaction was within the preview frame so the area of activation of every "3D-Touch submission action" is the same.
